### ISSUE TITLE
preventing overflow of disgusting content

### ISSUE THREAD

**Issue by HK53** [Chocobozzz/PeerTube#3768]
Dear all,
about a year ago I learned about PeerTube, for instance on https://switching.software/replace/youtube/ it is described as an 'Ethical, easy-to-use and privacy-conscious alternative' to Youtube. I have started using a PeerTube instance to share some homemade music video's to the world.

But now I have decided to pull out. That is because the federated video world of PeerTube is overflowing with disgusted content, at least, when you look at the Trending video's. Men slaughter, rape, mutilations, sex abuse, and the like. You have to look hard to find something respectful to humans.

Now, I do not have to look at Trending video's. So, problem solved? No, the viewers of my video's get this awful rubbish presented in the side bar, right next to my video's.

YouTube with its filter bubbles is at least trying to present the viewer with mostly video's that more or less resemble the area of interest of the viewed video.

My feature request is that I can choose that the side bar with video suggestions is not shown next to my video's. In order that when I invite someone to view my playlist or some particular video of mine, in the user experience there will be no other content in any way.

If this feature request can be realized soon, I might stay. If it will be realized in the future I might come back.

The present situation is awful. I am aware that this is not because of the PeerTube software. It is because of the way people are using PeerTube instances. But I wonder, is it really the goal of the PeerTube developers to create a video world where the worst images can freely be shared.

Thanks for your attention.

Kind regards,

HK

**Comment by Chocobozzz** [comment]
Hello,

Unfortunately some admins do not moderate their instance so hiding the "other videos" tab won't help. 

I suggest you to move to another instance that correctly moderates user content, and do not federate with unknown/weird PeerTube instances. You can use https://joinpeertube.org/instances#instances-list to choose this kind of instance. Then you can also check for the instance terms/code of conduct in the about page (for example: https://peertube.social/about/instance).

**Comment by HK53** [comment]
Thank you, @Chocobozzz , for the reaction. But it leaves me with questions. I have been looking at the PeerTube settings but could not find an option to hide the video suggestions next to my video. I have been perusing the instances list several times, but have found no clues to admins mitigating offending content from the Federation. I only see an option to hide 'adult content', but my observation is that it is much more than that. PeerTube/Fediverse/Framasoft has a very serious problem here, that needs to be solved before the services can be advised to a general public, young and old.

I don't mind discussing this elsewhere, but would no know where.

**Comment by rigelk** [comment]
@HK53 maybe just open a feature request for admins/users to toggle video suggestions?

**Comment by HK53** [comment]
Thank you @rigelk  for your reaction. But, is that not what I am doing in my initial post? With (maybe too much) explanation about why I want such a toggle? As for my it should be part of the settings of the videos I manage. So, it is not for the viewer to toggle, but for the creator. My desire for my viewers is to have a 'feel safe' view of my videos. Thanks again for your reaction.

**Comment by rigelk** [comment]
Not really. To be honest I got more insight in what the feature you want looks like in your last post than in the opening post of this issue, which buried the feature request in a single §. I invite you to fill the [feature request template](https://github.com/Chocobozzz/PeerTube/issues/new?assignees=&labels=&template=30-feature-request.md) with a descriptive title and open there a discussion about where the toggle should be/who should have control on it, etc.

**Comment by HK53** [comment]
Thank you, @rigelk . I have done so right away, see https://github.com/Chocobozzz/PeerTube/issues/3771 . Are you ok with the description?

**Comment by tiotrom** [comment]
This is very difficult to handle...it all depends on what other users post and moderation. For now, till Peertube becomes more mature and saner people start using it and add more and more saner videos, we removed this suggested videos thing entirely from our instance https://videos.trom.tf/ - I got annoyed that our hundreds of videos that we struggle to produce about science and the like, had all kinds of crappy videos suggested in the sidebar. Perhaps a solution would be to only show suggested videos from local videos or from the same channel.

**Comment by HK53** [comment]
Thank you, @tiotrom . I did not know that a local administrator had the freedom to remove parts of the interface. Removing the 'suggested videos' entirely is a good idea. I looked at your PeerTube instance. De startpage is the Trending, which still contains some rubbish videos. Can you make the 'Local videos' the start page, and remove  all the menu options which show unmoderated content from the Peertube Fediverse?

As far as I am concerned, waiting for the PeerTube community to become saner and mature is fruitless. We live in a crooked world, where every free venue for distribution will be misused. Rigorous moderation is the only option.

**Comment by tiotrom** [comment]
I removed them with a simple CSS line in the Admin - Advance. I did make the Local Videos the default page but is this bug with Peertube that it still displays the trending when you first visit the instance...I was looking for a way to make a channel at my choice, the default landing page, or the About. But right now there is no such thing. Others have asked for that too.

And yes, I can remove the left side menu too, but then it defeats the purpose of Peertube entirely....
And to moderate an instance is not easy. We will not moderate ours, since it is what we call as trade-free (trade-free.org), but we want to make sure that if sane people host their videos there, they can share their channel or videos without the sidebar videos present, so they share their own videos and that's it. Maybe....maybe...we will also remove the sidebar with "discover, trending" and such... I really get you. We too host hundreds of videos on our instance and we don't want them to be polluted...
